raw_description/urdf/vacuum_gripper/gripper.urdf.xacro

```diff
@@ -38,9 +38,9 @@
             </material>
         </visual>
         <collision>
-            <origin xyz="0.01 0.04 -0.048" rpy="0 0 0"/>
+            <origin xyz="0 0 -0.048" rpy="0 0 0"/>
             <geometry>
-                <box size="0.13 0.14 0.071"/>
+                <mesh filename="package://raw_description/meshes/vacuum_gripper/vacuum_gripper.stl"/>
```

Comment:
using a `box` instead of the `gripper.stl` in the `collision`-tag was done on purpose in https://github.com/ipa320/cob_common/pull/232